cmd/subnetcmd/list.go

```diff
@@ -51,7 +52,7 @@ func listGenesis(cmd *cobra.Command, args []string) error {
 
 	rows := subnetMatrix{}
 	// append a second "header" row for the networks
-	rows = append(rows, []string{"", "", "", "", "Local", "Fuji", "Mainnet"})
+	rows = append(rows, []string{"", "", "", "", "", "Local", "Fuji", "Mainnet"})
```

Comment:
do we need to print which is the repo? version? anything like that?

Reply:
Let's add that info in describe instead of list. The table in list is already getting a bit big

Reply:
#239